### DISCUSSION TITLE
Troubleshooting Issues with MKDocs Deployment

### DISCUSSION THREAD

**Discussion by KeithIMyers** [squidfunk/mkdocs-material#6309] [Q&A] [answered]
Good Afternoon,
I have finally finished migrating my entire website off of WordPress, onto MkDocs Material and I love it. There are a few strange problems with my new website that I am trying to troubleshoot and would love some help to troubleshoot/resolve the problems. I am sure this is a issue with my setup and not the product itself as I have not seen any other similar reports.

## Mobile Device Menu

When on a mobile browser (or a desktop browser that has been resized to give the mobile view). You need to select "Home" before it will show the navigation menu 

### Steps to Replicate
1) Open https://kmyers.me on a mobile phone or desktop browser resized to trigger the mobile view and tap on the Hamburger Menu on the top left corner of the screen.
2) Notice there is only 1 item "Home", Tapping "Home" will show the intended menu

## Blog Pagination 

When using the blog plugin, the pagination works perfectly when clicking on a page number thats higher than the current page but on lower pages, it will always fail to /blog/

### Steps To Replicate
1) Open a browser to https://kmyers.me/blog/ and navigate to the bottom. Click on page 2, than page 3, than page 4 (or any higher page, it wont matter).
2) Now try to click on any lower number page (except page 1) and you will be sent back to the root of the blog (page 1/https://kmyers.me/blog/) regardless of which page you select.

**Comment by mjtobias** [comment]
@KeithIMyers Was going to take a look at the pagination issue, verified that I can reproduce the issue and was going to suggest that it is a bug.  Looks like someone did that and seems like it is actually a bug per https://github.com/squidfunk/mkdocs-material/issues/6365

I also have the issue with the mobile device menu and am not sure at this time if it is a bug or not.

**Reply by KeithIMyers** [reply]
I was trying to avoid opening up a bug report, I had a feeling the issue may have been with my specific setup but I am glad someone else confirmed. I have not had time to troubleshoot the menu issue further

**Reply by mjtobias** [reply, marked as answer]
@KeithIMyers Think I have figured out the issue with the menu, I was also experiencing this.  I noticed that on a material for mkdocs site, https://github.com/twhite96/tifflabs/blob/main/mkdocs.yml, it wasn't the same issue.  Just tested this on my site and it does fix it.

In your `mkdocs.yml` file, under the navigation section, I'd be willing to be you have something similar to this:
```
nav:
  - Home:
      - index.md
   ... rest of your nav
```
This is telling mkdocs to create a section called Home with a page of Index.  I think what you want is this:
```

nav:
  - Home: index.md
   ... rest of your nav
```

**Reply by KeithIMyers** [reply]
That did it. Thanks
